app-text/cherrytree/cherrytree-9999.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6..8} )
+
+inherit cmake python-any-r1 xdg-utils
+
+DESCRIPTION="A hierarchical note taking application (C++ version)"
+HOMEPAGE="https://www.giuspen.com/cherrytree https://github.com/giuspen/cherrytree"
+
+if [[ ${PV} == *9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/giuspen/cherrytree"
+	S="${WORKDIR}/${P}/future"
+else
+	COMMIT="4e66c82c108bdfb56983d2c3614d88cc61f5179e" # 20200520
+	SRC_URI="https://github.com/giuspen/cherrytree/archive/${COMMIT}.tar.gz -> ${P}.tar.gz"
+
+	KEYWORDS="~amd64 ~x86"
+
+	S="${WORKDIR}/${PN}-${COMMIT}/future"
+fi
+
+# GPL-3 — future/src/ct (CherryTree)
+# LGPL-2.1 — future/src/7za (7zip)
+# MIT — future/src/fmt (libfmt)
+LICENSE="GPL-3 LGPL-2.1 MIT"
+SLOT="0"
+IUSE="nls test"
+
+RESTRICT="!test? ( test )"
+
+RDEPEND="
+	app-text/gspell
+	dev-libs/glib:2
+	>=dev-cpp/glibmm-2.64.2:2
+	dev-cpp/gtkmm:3.0
+	dev-cpp/gtksourceviewmm:3.0
+	dev-libs/libxml2:2
+	dev-cpp/libxmlpp:2.6
+	dev-cpp/pangomm
+	dev-db/sqlite:3"
+DEPEND="${RDEPEND}
+	$(python_gen_any_dep 'dev-python/lxml[${PYTHON_USEDEP}]')"
+BDEPEND="${PYTHON_DEPS}
+	virtual/pkgconfig
+	nls? (
+		dev-util/intltool
+		sys-devel/gettext
+	)
+	test? ( dev-util/cpputest )"
+
+PATCHES=(
+	"${FILESDIR}/cherrytree-0.99.0_p20200520_make_nls_optional.patch"
```

Comment:
did you think it through if it is really worth that patch? I mean, if you submit it upstream and they accept it, well done, but otherwise this is additional maintenance whenever upstream code changes in that area just for nls.